blaze-server/src/main/scala/org/http4s/server/blaze/WebSocketSupport.scala

```diff
@@ -52,14 +52,19 @@ private[blaze] trait WebSocketSupport[F[_]] extends Http1ServerStage[F] {
             hdrs.foreach {
               case (k, v) => sb.append(k).append(": ").append(v).append('\r').append('\n')
             }
+
+            val wsContext = ws.get
```

Comment:
This is not your fault, but it's something we can clean up.  Can we do a match on the `ws` Option instead of `if (ws.isDefined) { ???; ws.get ; ??? } else { ??? }`?

Reply:
sure, I'll change that